test/module/irohad/execution/command_validate_execute_test.cpp

```diff
@@ -1245,7 +1243,8 @@ class TransferAssetTest : public CommandValidateExecuteTest {
                            .balance(*balance)
                            .build());
 
-    role_permissions = {toString(Role::kTransfer), toString(Role::kReceive)};
+    role_permissions.set(Role::kTransfer);
```

Comment:
Is it possible to use initalizer list constructor here? Like `role_permissions = {Role::kTransfer, Role::kReceive};` Same for other usages and tests